### DISCUSSION TITLE
Welcome to Attendance-management-system Discussions!

### DISCUSSION THREAD

**Discussion by karnwalaryan07** [karnwalaryan07/Attendance-management-system#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by karnwalaryan07** [comment]
Hi guys, please discuss everything here from now onwards.

**Comment by karnwalaryan07** [comment]
I am briefing those who were not present in the meeting today (01-11-23) . 
  We decided to make the used case in which we will be having 3 portals for the login part i.e 3 options which will be student, teacher 
  and 
  Admin. 
  when we click on the Admin portal, then a window/page should open where we will be having 2 option: (a) student  (b)teacher 
- If the admin click on the teacher option then he will be able to edit all the details present in the teachers page/window and same goes 
  for the student. 
- we all know that Student should only be able to view the attendance and teacher should be able to edit it. 
- each portal/option will have an interface for user login and password ...

**Comment by karnwalaryan07** [comment]
So let's meet tomorrow at 4 p.m after the class. 
Everyone must be present as we have to show the demo working piece of the project on Saturday.

**Comment by diyatomar04** [comment]
okay..A suggestion. I think we should make subteams in the group so that we can complete the work on time.

**Comment by diyatomar04** [comment]
For these 3 portals we can divide the team into 3(in 2 3 3 sub-teams ). Front end and backend part we can divide our team into 2 .

**Comment by Bhavyajain0604** [comment]
Diya we cant divide sub teams bcuz the main motive of the project is how we can all cordinate togeather as a 8 members team. if we divided so sir would say you all didnt get why i have grouped you in 8 members.
we can divide the work amonge ourselves and take care of others works too but if we divied in sub teams so particular team will get restricted to their part only and it will reflect negative impact in coversation among team and btw today i was not there so i don't know what you all have discussed so far tommorow we are having meeting there we will discuss more deeply regarding this but by that time we can't divide in sub teams

**Comment by karnwalaryan07** [comment]
Being unbias...we'll ask for everyone's suggestion tomorrow, for creating sub-teams. 
Me, as an individual agree with Bhavya's point here as I also did not felt a need to create sub-teams.

**Comment by karnwalaryan07** [comment]
But still keeping her point in mind...we'll go with everyone's decision.

**Comment by diyatomar04** [comment]
I am not saying that we will restrict the work in sub teams.  Basically what i meant is that we can divide the work.

**Reply by karnwalaryan07** [reply]
Don't worry, we'll surely do it tomorrow without subdividing our team. we'll divide the sub-parts of our project.

**Comment by diyatomar04** [comment]
Because if all 8 people will work only on a single part of code , that will be a waste of time. So we can create sub-teams and divide the work. Obviously we will help each other in their part too, if needed

**Comment by diyatomar04** [comment]
guys we have to make use case using kanban , then we have code to complete(admin part) and a breif intro on git to our topic . We have a lot to do . Lets be quick

**Comment by karnwalaryan07** [comment]
so let's meet tomorrow at 11?

**Comment by Sairanjan-Subudhi** [comment]
Sorry I won't be able to attend the meeting as I'm in the organising committee for tomorrow's event and my day is packed.

**Comment by diyatomar04** [comment]
before meet plz go through **Kaanban** first

**Comment by Sairanjan-Subudhi** [comment]
> I am not saying that we will restrict the work in sub teams.  Basically what i meant is that we can divide the work.

Yeah if that's the case then it's fine👍

**Comment by Sairanjan-Subudhi** [comment]
> Because if all 8 people will work only on a single part of code , that will be a waste of time. So we can create sub-teams and divide the work. Obviously we will help each other in their part too, if needed

Hn help will be required as everyone is not well known with the coding part.

**Reply by karnwalaryan07** [reply]
This will be discussed in the offline meeting. If there will be a need, it will surely be done.

**Comment by karnwalaryan07** [comment]
Till then let's not discuss about sub-teams until five members are active.

**Comment by karnwalaryan07** [comment]
Goodnight !!

**Comment by diyatomar04** [comment]
![WhatsApp Image 2023-11-02 at 16 05 44_dfd1ce20](https://github.com/karnwalaryan07/Attendance-management-system/assets/117175207/2969891e-54aa-40cd-b831-c46095e75976)

**Comment by diyatomar04** [comment]
Rough diagrammatic approach

**Comment by diyatomar04** [comment]
![Uploading WhatsApp Image 2023-11-04 at 10.30.51_a346f8ba.jpg…]()

**Comment by saumyasehgal** [comment]
![usecase](https://github.com/karnwalaryan07/Attendance-management-system/assets/143324900/a41b1a65-ed70-4206-a703-228a20996fe4)
USECASE DIAGRAM

**Comment by diyatomar04** [comment]
![WhatsApp Image 2023-11-04 at 10 30 51_944bd034](https://github.com/karnwalaryan07/Attendance-management-system/assets/117175207/52b48f4c-8d82-4e13-80a1-0e46931a561c)
final Use Case

**Comment by Bhavyajain0604** [comment]
![WhatsApp Image 2023-11-04 at 11 36 20_e931be39](https://github.com/karnwalaryan07/Attendance-management-system/assets/131136742/8cef21c6-cd0a-4f83-9bf9-d9556011b269)

Guys like this our whole project will be made

**Comment by diyatomar04** [comment]
Yogesh has almost completed the frontend part..Me and Bhavya are left with final desiging of that part.

**Comment by karnwalaryan07** [comment]
backend is also completed upto some extent

**Comment by karnwalaryan07** [comment]
@Bhavyajain0604 @diyatomar04 Design the pages as soon as possible and send the codes to us for further changes

**Comment by karnwalaryan07** [comment]
so, everything is almost completed guys and some final touchups are left as per i know along with the commits on the github...The code is in running condition and pages are looking quite descent. Good work!!

**Comment by diyatomar04** [comment]
Well just one page is left if i am not wrong

**Comment by diyatomar04** [comment]
![WhatsApp Image 2023-12-06 at 02 22 24_d928b894](https://github.com/karnwalaryan07/Attendance-management-system/assets/117175207/c1fd8051-2921-4c86-b6d5-5b5456a609a6)
I have finalized this page . If you want to change anything let me know

**Comment by Bhavyajain0604** [comment]
![pic](https://github.com/karnwalaryan07/Attendance-management-system/assets/131136742/276a6ec0-32d7-40a5-91f4-b021a21e7861)
Guyss I am thinking that we should add this pic in background what you say about it

**Comment by Sairanjan-Subudhi** [comment]
Yes that's looking good👍

**Comment by Bhavyajain0604** [comment]
![image](https://github.com/karnwalaryan07/Attendance-management-system/assets/131136742/a5dac339-6894-4175-b855-0fcbf082ffad)
It is looking like that if you want any changes let me know asap!!

**Comment by diyatomar04** [comment]
Guys can u help and find similar pictures

**Comment by Bhavyajain0604** [comment]
Yes diya we are looking for the same

**Comment by diyatomar04** [comment]
![Uploading image.png…]()
